app/components/about-me.tsx

```diff
@@ -4,7 +4,7 @@ import { ContentContainer } from "@/components/content-container";
 import { InlineImageContainer } from "@/components/inline-image-container";
 import { Paragraph } from "@/components/paragraph";
 
```

Comment:
Converted .jpgs to .webp. Still some optimization to be done around images but it's not really a major issue.